Sources/TerminalAPIKit/Models/Operation.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Operation.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public final class Operation: Codable {
+
+    /// The type of request after which the waiting screen shows: InputRequest, DisplayRequest, and Payment
+    public let type: OperationType
+    
+    /// For now, the Operation.Variant can only have WaitingScreen as value. This shows the waiting screen animation on the terminal after every request until you end the session.
+    public let variant: OperationVariant
```

Comment:
we probably don't need to mention the options here, since the type is now an enum, and the possible values will be restricted to that. given that the `OperationVariant` enum contains only one case, it should be logical that it only supports one variant.

Reply:
[Adyen's docs](https://docs.adyen.com/point-of-sale/shopper-engagement/create-session/#start-session) do not go into any detail on this, and your comment here doesn't explicitly say what you want. Can you tell me explicitly what you want here for this comment? Otherwise I'll just be guessing, and we will just keep going back and forth.

Reply:
Latest commit should handle